test/test-applications/azure-functions/Samples.AzureFunctions.AllTriggers/AllTriggers.cs

```diff
@@ -0,0 +1,161 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using Azure.Storage.Blobs;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Azure.Documents;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+
+namespace Samples.AzureFunctions.AllTriggers
+{
+	public static class AllTriggers
+	{
+		private static readonly HttpClient FunctionHttpClient = new HttpClient();
+		private const string EveryTenSeconds = "*/10 * * * * *";
+
+		[FunctionName("TriggerAllTimer")]
+		public static async Task TriggerAllTimer([TimerTrigger(EveryTenSeconds)] TimerInfo myTimer, ILogger log)
+		{
+			log.LogInformation($"C# Timer trigger function executed at: {DateTime.Now}");
+			await CallFunctionHttp("trigger");
+		}
```

Comment:
Are we able to run integration tests for the timer/http triggers, as they don't require any Azure infra?

Reply:
Yes, I plan to include those in a follow up, but I will try to get them in today.

Reply:
I have not successfully run the functions host in our integration test framework yet, but the integration is running on several sample apps and I will monitor them very closely and continue work on the automation.